### ISSUE TITLE
exec_context_dpu  - null values for context

### ISSUE THREAD

**Issue by tomas-knap** [mff-uk/odcs#677]
Is it ok that the values binding the table to context are null? I mean column exec_context_pipeline_id

![screen shot 2013-10-18 at 2 36 52 pm](https://f.cloud.github.com/assets/3014917/1360239/fc5f8fec-37f1-11e3-884c-e58b747aabc3.png)

**Comment by janvojt** [comment]
I think Petr might shed some light and answer this question -> I am reassigning.

P.S.: Is this a bug? So far I suspect rather a discussion...

**Comment by skodapetr** [comment]
Table exec_context_dpu is for ProcessingUnitInfo class. The exec_context_pipeline_id corresponds with mapping from ExecutionContextInfo. So it here is null .. then we have information about execution for single DPU but we are missing the information about execution itself.    This should not happen. 

Honza can this be similar like problem with ExecutionContextInfo mapping problem (we solve that by using 1:1 mapping).

But Honza is right .. that is should cause no harm .. just spam database.

**Comment by skodapetr** [comment]
Honza do you have idea how those can be created?

**Comment by janvojt** [comment]
No, this is completely managed by backend. Maybe we can investigate if it can be reproduced in current version with a new pipeline run.
